doc/manual/manual.xml

```diff
@@ -63,583 +63,6 @@
       <xi:include href="machine-options.xml" />
```

Comment:
Should the virtualbox documentation be here still ?
I think since it's moving away to its own repository, the documentation should be removed from here as well ?

Reply:
@AmineChikhaoui, yes, I think you're right that the virtualbox documentation should be moved to the virtualbox repository.  I've removed the virtualbox doc section from overview.xml in #https://github.com/NixOS/nixops/pull/1179/commits/40eb96a28b2a9a96f0aa0217244817c356f04a83

Reply:
Thanks !